stdlib/public/core/Tuple.swift.gyb

```diff
@@ -23,6 +23,82 @@ comparableOperators = [
 
 }%
 
+/// Returns a Boolean value indicating whether the corresponding components of
+/// two tuples are equal.
+///
+/// All arity zero tuples are equal.
+///
+/// - Parameters:
+///   - lhs: An empty tuple.
+///   - rhs: An empty tuple.
+public func ==(lhs: (), rhs: ()) -> Bool {
+  return true
+}
+
+/// Returns a Boolean value indicating whether any corresponding components of
+/// the two tuples are not equal.
+///
+/// Two tuples each of arity zero are never unequal
```

Comment:
'never unequal' is a bit hard to understand (and there is a period missing ;-)). I'd go with the same note as for the `==`.